.github/workflows/deployment-test.yml

```
@@ -27,9 +27,9 @@ jobs:
   deployment-test:
     needs: pre_check
     if: needs.pre_check.outputs.should_run == 'true'
-    runs-on: ubuntu-18.04 # trusty
+    runs-on: ubuntu-22.04 # trusty
```

I believe this will break. I think we need to update the deployment test scripts first. I'll add the `force:integration` label on this one.

---

As I recalled, we're stuck on Ubuntu 18 because of some incompatibility in the ansible based deployment. @michaelfig any idea? In the mean time I recommend backing out this specific change.

---

I'm fine with at least letting CI try to run it.  But if it works, please do update the Ubuntu codename (22.04 is not "trusty"):

```suggestion
    runs-on: ubuntu-22.04 # jammy
```

---

Indeed this breaks deployment-test. It didn't fail but it [timed out after 6hrs](https://github.com/Agoric/agoric-sdk/actions/runs/3992855547/jobs/6849102408). Errors point to Ansible as predicted.

I've filed https://github.com/Agoric/agoric-sdk/issues/6840 and I'll back at that change from this PR. Force push coming.